src/utils/known-tokens.js

```diff
@@ -0,0 +1,10 @@
+import env from '../environment'
+import { getNetworkName } from '../lib/web3-utils'
+
+export const KNOWN_DAI_ADDRESS_BY_ENV = {
```

Comment:
What do you think about making this generic? so `Known_tokens_By_Env` and not just with the address maybe also with the token id, decimals and symbol ?

Reply:
Yep, we can make it generic and add the `ANT` as well 👍